External/FEXCore/unittests/Emitter/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+if (ENABLE_VIXL_DISASSEMBLER)
```

Comment:
Since this check is hidden in a third-level CMakeLists.txt, maybe add a warning about emitter tests being disabled if tests are enabled but vixl is not? When testing this PR, it took me a couple of minutes to figure out why the emitter tests weren't building for me.

Reply:
Resolved.